integration-tests/change-base-dir/openapi-definitions/petstore/petstore.json

```diff
@@ -0,0 +1,1229 @@
+{
+  "openapi": "3.0.2",
```

Comment:
Instead of this complex OpenAPI, can we please use this one? https://github.com/quarkiverse/quarkus-openapi-generator/blob/main/integration-tests/simple/src/main/openapi/quarkus-simple-openapi.yaml

It's simpler to configure, and not much to parse so that the test will run slightly faster.

Reply:
yup, sure